Users/Install.rst

```diff
@@ -285,6 +285,13 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Shell-autocompletion support
+----------------------------
+
+If you are a bash/zsh user , checkout the
+`guide <http://docs.coala.io/en/latest/Users/Shell_Autocompletion.html>`
+for setting-up tab-completion for coala arguements and bear names.
```

Comment:
`tab-completion` -> `autocompletion` (make this change in the commit body and shortlog as well)